### ISSUE TITLE
[New Feature Request]: Visual Studio Palette

### ISSUE THREAD

**Issue by Wagnerp** [Wagnerp/Krypton-NET-5.470#235]
A Visual Studio 201x theme!

**Comment by Smurf-IV** [comment]
Because you have left it as **x**, then this could be controversial, because event the blue theme has changed for 4 out of the 5 versions, there there are the other colour themes to implement / use.

**Comment by Wagnerp** [comment]
Should I rename it to 2019? It came up on the Discord chat group.

**Comment by ZeyadAhmed** [comment]
2017 theme is decent and it's popular (I did not install 19 yet), Color value references from MSDN for VS 2017, if it seems complicated, I can get the RGBs of the important things.  but there is a problem in scrollbars. we need to find a way to change the scrollbar look according to a pallette not just for the built-in-krypton tools or we will get like this result (DarkUI used)
![image](https://user-images.githubusercontent.com/19334375/66245578-f9eb0c00-e70e-11e9-8ec5-31b34e76a18a.png)

**Comment by Wagnerp** [comment]
Hi @Smurf-IV & @ZeyadAhmed 

What I could do for the interim is to create a new `xml` theme file using the colours that @ZeyadAhmed has provided, until I can find out how the built-in palette system works. How does this sound?

**Comment by Smurf-IV** [comment]
@Wagnerp That's possible of course, but you will have to state what colour of the visual studio theme you are using as well (i.e. Dark(Black or is it grey?), Blue, etc.)

**Comment by Wagnerp** [comment]
@Smurf-IV Yes sure, will name the file `VisualStudio2015` + colour. The other route is to use the current `Office365PaletteBase` to make native themes.

**Comment by Wagnerp** [comment]
Hi @Smurf-IV & @ZeyadAhmed 

This is what I've come up with so far... 
[Visual Studio 2019 Dark.zip](https://github.com/Wagnerp/Krypton-NET-5.470/files/3701406/Visual.Studio.2019.Dark.zip)

**Comment by Wagnerp** [comment]
May need a few adjustments

![image](https://user-images.githubusercontent.com/949607/66477698-935c4a00-ea90-11e9-922e-d0ff982349ea.png)

**Comment by Wagnerp** [comment]
![Annotation 2019-10-12 191910](https://user-images.githubusercontent.com/949607/66705972-58bf0f80-ed25-11e9-893f-ff98aa1fd3dd.png)

**Comment by ZeyadAhmed** [comment]
Oh cool I have seen your reply on discord, It's good but the largest step will be when having a function to set the scrollbar theme according to the selected palette like "ApplyThemeToControl(MyListBox);" will be amazing!
